### DISCUSSION TITLE
Coverage analytics service

### DISCUSSION THREAD

**Discussion by MrBartusek** [TheDrone7/shieldbow#23] [Ideas]
Hey, we run unit tests on each commit and collect coverage but it is not the only way to check it is by browsing locally generated coverage reports. It would be great if coverage would be sent to some service like [codecov](https://about.codecov.io)

**Comment by TheDrone7** [comment]
Good idea, I've added it.

**Reply by MrBartusek** [reply]
Hey, i was thinking about something cloud-based like [codecov](https://about.codecov.io) so you don't need to spin up env to see coverage or see coverage changes on PRs
